src/saltext/vmware/modules/vmc_public_ip.py

```diff
@@ -107,3 +107,138 @@ def get(
         cert=cert,
         params=params,
     )
+
+
+def get_by_id(
+    hostname,
+    refresh_key,
+    authorization_host,
+    org_id,
+    sddc_id,
+    public_ip_id,
+    verify_ssl=True,
+    cert=None,
+):
+    """
+    Retrieves Public IPs from Given SDDC
```

Comment:
Doc for this is identical to the `get` function (which should also be `list`). Either this one or `get` should be changed.